test/features/schema_tests.js

```diff
@@ -34,9 +35,16 @@ parallel('Responses should conform to the provided JSON schema of the response',
 
     it('should expose valid OpenAPI spec', () => {
         return preq.get({ uri: `${server.config.baseURL()}/?spec` })
-        .then((res) => {
-            // TODO: validate the openapi spec!
-        });
+            .then((res) =>  {
+                return preq.post({uri: `http://online.swagger.io/validator/debug`,
```

Comment:
I think it's better to install a dev dependency to swagger-cli and use it to validate. Lett dependencies on internet and external service.

Reply:
One of my concerns with swagger-cli is that it uses swagger-parser which is using a deprecated dependency: https://github.com/APIDevTools/swagger-parser/pull/109

Reply:
hm. ok. Let's use the online one for now then and file a ticket to keep an eye on the swagger-cli thing.